### PR TITLE
Rework URL structure in preparation for Maths & Physics policy

### PR DESCRIPTION
**Pull request by robbiepaul**
In preparation for adding another policy (Maths & Physics) we need to have different URLs for each one. 

- Include the policy name as a parameter available the controller
- Only allow valid policies in the URL
- Include URL helpers in the asset pipeline as calling them statically from assets doesn't set the correct defaults

### CODE REVIEW

**Comment by robbiepaul** [comment]
[Rework URL structure in preparation for Maths & Physics](https://trello.com/c/RVCRHSHo/750-rework-url-structure-in-preparation-for-maths-physics)

File: config/routes.rb [hunk 1]
```diff
@@ -21,12 +21,14 @@
   get "/cookies", to: "static_pages#cookies", as: :cookies
   get "/accessibility_statement", to: "static_pages#accessibility_statement", as: :accessibility_statement
 
-  constraints slug: %r{#{PageSequence::SLUGS.join("|")}} do
-    resources :claims, only: [:new, :create, :show, :update], param: :slug, path: "/claim"
-  end
+  scope path: ":policy", defaults: {policy: "student-loans"}, constraints: {policy: %r{student-loans}} do
+    constraints slug: %r{#{PageSequence::SLUGS.join("|")}} do
+      resources :claims, only: [:new, :create, :show, :update], param: :slug, path: "/"
```
Comment (pezholio) [inline config/routes.rb]: Is it worth sending the policy to the controller? (even though we might not use it yet), so we can set the eligibility model at a later date?

Comment (robbiepaul) [reply config/routes.rb]: Yeah the policy is available in the controller via `params[:policy]` at the moment. Were you thinking of something else?

Comment (pezholio) [reply config/routes.rb]: Oh yeah, sorry, I missed that 🤦‍♂

**Comment by robbiepaul** [comment]
It's worth mentioning `/student-loans` is currently a 404, not sure if we need to tackle that now or as part of a story later

**Comment by pezholio** [comment]
That might be worth doing after #477 goes in, so we can redirect `/student-loans` to the `new` action, but that might need a bit more thought

**Pull request merged**